graph/telemetry/istio/appender/kiali_annotation.go

```diff
@@ -0,0 +1,69 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/models/annotation"
+)
+
+const AnnotationAppenderName = "kialiAnnotation"
```

Comment:
This is similar to the Sidecars appender and could feasibly be combined into that appender, but there are a few differences and I'm not opposed to adding a new appender specifically to handle these config overrides.   As mentioned above, I do think the name should change to better reflect what it is doing.  Assuming the proposed name change:

`const HealthConfigAppenderName = "healthConfig"`